packages/api/orm/entity/Clin.ts

```diff
@@ -0,0 +1,51 @@
+import { BaseEntity } from "./BaseEntity";
+import { Column, Entity, ManyToOne } from "typeorm";
+import { TaskOrder } from "./TaskOrder";
+
+@Entity()
+export class Clin extends BaseEntity {
+  @Column({
+    type: String,
```

Comment:
A good question.  I don't have a good answer, and have wondered the same.  I think the source may have been some example.  It's probably better to remove the type specification from the column options and rely on the conversion of the TypeScript type to the compatible postgres type in these cases.  I'll look more deeply at this and either explain or replace.

Reply:
I know i'm late and it seems like the `type` will be removed, but in case we want to keep it `varchar` might be the `type` that best fits according to the [Postgres docs](https://www.postgresql.org/docs/9.3/datatype-character.html).

Reply:
For simple TypeScript `string` fields, I removed the type from the column options.  For two that are arrays of strings, I changed the type in the column options to `varchar`.

Updated in f14ec95b8c703c52846010133b574130fa777f79